packages/lab/src/avatar/Avatar.tsx

```diff
@@ -1,74 +1,88 @@
-import { makePrefixer, useId } from "@salt-ds/core";
+import { makePrefixer } from "@salt-ds/core";
+import { UserIcon } from "@salt-ds/icons";
 import { clsx } from "clsx";
-import { forwardRef, HTMLAttributes, ImgHTMLAttributes } from "react";
-import { classBase } from "./internal/constants";
-import { DefaultAvatar } from "./internal/DefaultAvatar";
+import { forwardRef, HTMLAttributes, ReactNode } from "react";
 import { useLoaded } from "./internal/useLoaded";
 
 import "./Avatar.css";
 
+export type InitialsGetter = (name?: string) => string;
+
 export interface AvatarProps extends HTMLAttributes<HTMLDivElement> {
-  alt?: string;
-  id?: string;
-  imgProps?: ImgHTMLAttributes<HTMLImageElement>;
-  size?: "small" | "medium" | "large";
-  sizes?: string;
+  /**
+   * The name that Avatar represents.
+   */
+  name?: string;
+  /**
+   * Defines the function that gets initials. Default is capital first letter of each separate word in name.
+   * If a function is not passed or returns undefined, Avatar will default to Icon.
+   */
+  initialsGetter?: InitialsGetter;
```

Comment:
We should think about the name of this

Reply:
`initialsReducer` `nameShortener` `initialsExtractor` ? 🤷‍♀️ any suggestions?